connectors/v2/youtube.js

```diff
@@ -51,12 +51,95 @@ Connector.getArtistTrack = function () {
 		track = text.substr(separator.index + separator.length);
 	}
 
-	/**
-	* Clean non-informative garbage from title
-	*/
+	return {
+		artist: cleanseArtist(artist),
+		track: cleanseTrack(track)
+	};
+};
+
+var timestampPattern = '[0-9]{0,2}:*[0-9]{1,2}:[0-9]{2}';
+var timestampRegex = new RegExp(timestampPattern,'gi');
+
+Connector.getPlaylist = function() {
+	var playlist = [];
+	var $containers = $('#eow-description, .comment-text-content');
+
+	var i = 0;
+	var found = false;
+	while($containers.get(i) && !found) {
+		var $container = $($containers.get(i));
+		var potentialPlaylist = [];
+		if(timestampRegex.test($container.html())) {
+			var potentialTracks = $container.html().split(/\r\n|\r|\n|<br>/g);
+			potentialPlaylist = parsePlaylist(potentialTracks);
+		}
+		if (potentialPlaylist.length > 1) {
+			playlist = potentialPlaylist;
+			found = true;
+		} else {
+			i++;
+		}
+	}
+
+	if(playlist.length <= 1) { return; }
```

Comment:
Open question here: should the threshold for 'playlist' be higher than 1? 
* Maybe it should be 3+ for comments, 2+ for descriptions?